### PR TITLE
Removed old reference to `vendor` bundle and external jQuery in webpack

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Vendor entry was only used to load bootstrap dependencies (required for DataTables) and is no longer required.  Also there was a legacy reference to jQuery in the webpack.config which was removed.  Finally, base.html template was removed as it was no longer required (relative sections moved into page.html)

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~~* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~~
~~* [ ] Tests added (or description of how to test) for any new features.~~
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

File: src/main/webapp/resources/css/app.css [hunk 1]
```diff
@@ -91,42 +91,6 @@ body > .ant-layout {
...
-    }
-
-    & :not(.active) a {
-      color: #717f8c !important;
-      background-color: transparent !important;
-    }
-  }
-}
-
-label.ant-checkbox-wrapper {
```
Comment (ericenns) [inline src/main/webapp/resources/css/app.css:126]: Was this mistakenly removed, this is set for `.ant-checkbox-wrapper`

File: src/main/webapp/resources/js/components/Header/MainNavigation/MainNavigation.css [hunk 2]
```diff
@@ -39,7 +39,7 @@
 
 .main-navigation .ant-menu-title-content a {
   font-weight: 600;
-  font-size: 1.5rem;
+  font-size: 1.2rem;
```
Comment (ericenns) [inline src/main/webapp/resources/js/components/Header/MainNavigation/MainNavigation.css:42]: Why are we reducing the main nav font size here?

**Review by ericenns — CHANGES_REQUESTED** [review]
A few comments, but otherwise looking good.

File: src/main/webapp/resources/js/components/Header/MainNavigation/MainNavigation.css (hunk 2 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/components/Header/MainNavigation/MainNavigation.css:42]: When I removed bootstrap, this font in the nav blew up.  This just put it back in check

File: src/main/webapp/resources/css/app.css (hunk 1 above)
Comment (joshsadam) [reply src/main/webapp/resources/css/app.css:126]: Was there to override bootstraps checkbox css

File: src/main/webapp/resources/js/components/Header/MainNavigation/MainNavigation.css (hunk 2 above)
Comment (ericenns) [reply src/main/webapp/resources/js/components/Header/MainNavigation/MainNavigation.css:42]: Ok weird, glad to know that it is the same size pre PR

**Review by ericenns — APPROVED** [review]
👍

**Pull request merged**